### ISSUE TITLE
 In address validation TOS 1ZF-/2ZF

### ISSUE THREAD

**Issue by shivakrishna-vcare** [shivakrishna-vcare/OrangeScrum#177]
<p>is valid but while submitting the final carrier request carrier says it's invalid TOS. Enroll ID - A8205<br />for the same account in address validation 28F- is invalid TOS but for the final carrier request it accepts the same and carrier respons is positive.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>Please move below revision to qa and demo</p>
<p>/home/generic/Activation/homephone_review/homephone_new_enrollment_ajax.php -m "#940: In address validation TOS 1ZF-/2ZF"</p>
<p>Sending generic/Activation/homephone_review/function.php<br />Sending generic/Activation/homephone_review/home_phone_new_enrollment.php<br />Sending generic/Activation/homephone_review/homephone_new_enrollment_ajax.php<br />Sending generic/Activation/homephone_review/review_home_phone_feature.php<br />Transmitting file data ....done<br />Committing transaction...<br />Committed revision 78121.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>78121 &gt; QA 78123 &gt; DEMO 78124</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>Please move below revision to qa and demo</p>
<p>Sending generic/Activation/homephone_review/review_home_phone_feature.php<br />Transmitting file data .done<br />Committing transaction...<br />Committed revision 78202.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>78202 &gt; QA 78212 &gt; Demo 78213</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>Please move below revision to qa and demo</p>
<p>Sending generic/Activation/homephone_review/homephone_new_enrollment_ajax.php<br />Sending generic/Activation/homephone_review/review_home_phone_feature.php<br />Transmitting file data ..done<br />Committing transaction...<br />Committed revision 78294.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>78294 &gt; QA 78296 &gt; Demo 78297</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>Please move below revision to qa and demo</p>
<p>Sending generic/Activation/homephone_review/homephone_new_enrollment_ajax.php<br />Transmitting file data .done<br />Committing transaction...<br />Committed revision 78396.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>78396 &gt; QA 78397 &gt; DEMO 78398</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;anshul.srivastava commented:<br/><p>Team,<br />Same has been verified on dev server, please find below details for reference.</p>
<p>Cathect<br />address_validation_xml<br />13594 -- No TOS<br />13595 -- Review</p>
<p>Enrollment ID : A179</p>
<p>CityCom<br />Enrollment ID: A1768<br />Address Validation --&gt; 2BF-</p>
<p>Orderfunction<br />ID 17784<br />ASID 27516</p>
<p>Case I:- CityCom (TOS updated in final order submission)<br />Initial review TOS 2ZF-<br />Final Order Submission TOS 2BF-<br />Orderfunction ID 17974<br />WHSL ASID 27582</p>
<p>Case II:- CityCom (TOS updated on detail page)<br />Initial review TOS 2BF- updated to 2ZF-<br />Orderfunction ID 27583<br />WHSL ASID 27583</p>
<p><br />-- Do the changes mention in green block. - Resolved<br />-- Default TOS fields should be hidden just like Service address form in Suspend. -- Not Done<br />-- Intially on detail page 2BF- was submitted but on final order function 2ZF- was sent, but in OrderFunction it's 2BF- -- Resolved</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;pushkar.kumar commented:<br/><p>&nbsp;</p>
<p>&nbsp;</p>
<p>78121 &gt; QA 78123 &gt; DEMO 78124 &gt; Production<br />78202 &gt; QA 78212 &gt; DEMO 78213 &gt; Production<br />78294 &gt; QA 78296 &gt; DEMO 78297 &gt; Production<br />78396 &gt; QA 78397 &gt; DEMO 78398 &gt; Production</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>78121 &gt; QA 78123 &gt; DEMO 78124 &gt; Production 78892<br />78202 &gt; QA 78212 &gt; DEMO 78213 &gt; Production updated<br />78294 &gt; QA 78296 &gt; DEMO 78297 &gt; Production updated<br />78396 &gt; QA 78397 &gt; DEMO 78398 &gt; Production updated<br />&nbsp;<br />&nbsp;<br />&nbsp;<br />&nbsp;</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;anshul.srivastava commented:<br/><p>Team,&nbsp;</p>
<p>Same has been verified with the label and details. refer attached screen shot for details.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;pushkar.kumar commented:<br/><p>&nbsp;</p>
<p>Closing this task.</p>
